tests/storage/data_table_test.cpp

```diff
@@ -69,5 +69,79 @@ TEST_F(DataTableTests, TransformTileGroupTest) {
   data_table->TransformTileGroup(0, theta);
 }
 
+/*
+ * SamplingForOptimizerTest - Tests basic table basic sampling
+ */
+TEST_F(DataTableTests, SamplingForOptimizerTest) {
```

Comment:
any test commands, such as expect_eq or expect_true, used in this test function?